### PR TITLE
Add new get state api, fix kqueue build error on Mac

### PR DESCRIPTION
**Pull request by akbarahmed**
Added a new API to get the current state of Dynomite.

```bash
curl http://localhost:22222/state/get_state
```

Also, fixed the kqueue build error on Mac. 

And lastly, updated `build.sh` to find the ssl libs on a Mac.

### CODE REVIEW

**Comment by ipapapa** [comment]
LGTM

File: src/dyn_stats.c [hunk 1]
```diff
@@ -1113,7 +1113,7 @@ parse_request(int sd, struct stats_cmd *st_cmd)
                 }
 
                 if (strncmp(reqline[1], "/state", 6) == 0) {
-                    log_debug(LOG_VERB, "Setting state - URL Parameters : %s", reqline[1]);
+                    log_debug(LOG_VERB, "Setting/Getting state - URL Parameters : %s", reqline[1]);
```
Comment (shailesh33) [inline src/dyn_stats.c]: can you also add this to the help command?

**Comment by akbarahmed** [comment]
Adding to command to help now. Will push in a minute.

**Comment by shailesh33** [comment]
Thank you @akbarahmed

**Pull request merged**